velox/functions/prestosql/GeometryFunctions.h

```diff
@@ -54,7 +54,7 @@ struct StGeometryFromTextFunction {
     GEOS_TRY(
         {
           geos::io::WKTReader reader;
-          geosGeometry = reader.read(wkt);
+          geosGeometry = reader.read(std::string(wkt));
```

Comment:
fixme: We don't want to do this. These strings are often large (MBs). The reader takes a `const std::string &`. Can we make an explicit cast to that? Or skip this one?